src/schemas/training.day.ts

```diff
@@ -3,8 +3,8 @@ import { TrainingDayDocument, TrainingDayModel } from '../types/db/training.day.
 
 const TrainingDaySchema = new Schema({
     name: { type: String, required: true },
-    excercises: [{
-        excerciseId: { type: String, required: true },
+    exercises: [{
+        exerciseId: { type: String, required: true },
```

Comment:
Soll das hier auf das Exercise Model referenzieren? Wenn ja dann fehlt da die Referenz
```suggestion
        exerciseId: { type: Schema.Types.ObjectId , ref: 'Exercise' },
```